### DISCUSSION TITLE
Check block doesn't work as expected

### DISCUSSION THREAD

**Discussion by starkgate** [postmanlabs/postman-flows#185] [Defect]
See picture below for an example. As you can see in the log, http statusCode is 200, yet the check doesn't let the request go through to the rest of the flowchart.

![vCZMog0YnN](https://user-images.githubusercontent.com/12874678/147222231-e7bffcd2-f119-4553-a8b5-7e00e4650a38.png)

**Comment by enochj** [comment]
I noticed this today.

**Comment by sapacheco** [comment]
As far as I know, the check block is for testing. Using the "condition" block with the exact same content as you have should work.

**Reply by topmangarbuja** [reply]
Yes. Condition block helps.

With the **Condition** block, can we use one condition block instead of multiple condition blocks which only checks `statusCode == 200` for all of the response of 'Send Request' blocks?
